docs/intelmqctl.md

```diff
@@ -210,13 +210,33 @@ The same as start and stop consecutively.
 ### reload
 The same as reload of every bot.
 
+### enable / disable
+The subcommands `enable` and `disable` are setting the according flags in the runtime.conf and thus make them part of the botnet.
```

Comment:
"set the corresponding flags"
"in runtime.conf"

delete "and thus make them part of the botnet"